accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml

```diff
@@ -90,10 +90,10 @@ OAUTH2_TOKEN_EXPIRES_IN:
   "implicit": 1200
 
 # The number of seconds after an access token is issued until it expires.
-ACCESS_TOKEN_EXPIRES_IN: 1200
+ACCESS_TOKEN_EXPIRES_IN: 12000
 
 # The number of seconds after a refresh token is issued until it expires.
-REFRESH_TOKEN_EXPIRES_IN: 2592000
+REFRESH_TOKEN_EXPIRES_IN: 25920000
```

Comment:
the default values in the fence default config are
```# The number of seconds after an access token is issued until it expires.
ACCESS_TOKEN_EXPIRES_IN: 1200

# The number of seconds after a refresh token is issued until it expires.
REFRESH_TOKEN_EXPIRES_IN: 2592000
```
why was this changed for NCT??

Reply:
In the PROD VM it was set to these values. The change was to match the public config with fence-config in PROD VM

Reply:
we should use the default values in QA and Prod 👍